iset.mm

```diff
@@ -105738,6 +105738,18 @@ which defines the set we are taking the supremum of must (a) be true at
       UKAVNNWDRUPRWEVOWFVPVRVSVTWG $.
   $}
 
+  $( Partial converse to ~ bezout .  Existence of a linear combination does not
+     set the GCD, but it does upper bound it.  (Contributed by Stefan O'Rear,
```

Comment:
The GCD divides any linear combination. (?)

Reply:
I'm pretty sure I like "upper bound" better. Granted it isn't an upper bound expressed quite as we usually state one, but it is an upper bound in a certain (multiplicative) sense, and makes for a better parallel to "set the GCD".

Also, this is not a place where iset.mm should diverge from set.mm.